spiffworkflow-backend/src/spiffworkflow_backend/routes/process_api_blueprint.py

```diff
@@ -185,7 +186,7 @@ def _process_data_fetcher(
                 status_code=400,
             )
 
-    process_data_value = bpmn_process_data.get(process_data_identifier)
+    process_data_value = bpmn_process_data.get("data_objects", bpmn_process_data).get(process_data_identifier)
```

Comment:
Running the get on the entire dictionary if data_objects is not found is the intent. Also since we just called get on bpmn_process_data then returning it as the default should not result in an error and should maintain old behavior which is the intent.